lib/src/cgcloud/lib/ec2.py

```diff
@@ -343,15 +343,24 @@ def spot_request_not_found( e ):
 
 
 def create_spot_instances( ec2, price, image_id, spec,
-                           num_instances=1, timeout=None, tentative=False ):
+                           num_instances=1, timeout=None, tentative=False, tag=''):
```

Comment:
Can we make this a bit more general, by allowing for a dictionary of tags to be passed in?